.github/workflows/test.yml

```diff
@@ -17,22 +17,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        # TODO: fix lsp-sonarlint-c++-reports-issues and include macos-latest
+        os: [ubuntu-latest]
         emacs-version:
           - 27.2
           - 28.2
           - 29.3
         experimental: [false]
-        include:
-          - os: ubuntu-latest
-            emacs-version: snapshot
-            experimental: true
-          - os: macos-latest
-            emacs-version: snapshot
-            experimental: true
-          - os: windows-latest
-            emacs-version: snapshot
-            experimental: true
+        # TODO: enable once emacs snapshot version is fixed
+        # see https://github.com/emacs-eask/cli/issues/224
+        # include:
+        #   - os: ubuntu-latest
+        #     emacs-version: snapshot
+        #     experimental: true
+        #   - os: macos-latest
+        #     emacs-version: snapshot
+        #     experimental: true
+        #   - os: windows-latest
+        #     emacs-version: snapshot
+        #     experimental: true
```

Comment:
We can just remove the `windows-latest` test.

Reply:
removed

Reply:
Sorry for the confusion. I mean comment out only the windows snapshot.

Reply:
The other two are problematic as well. as you can see, they take all the time in the world, even though they will eventually pass.

Reply:
Ah, okay. I guess we can just comment them all out.

Reply:
done 33f07a2e1c2e6a938fffe410795390ec2ee0242f